openedx_events/data.py

```diff
@@ -27,6 +27,16 @@ def _ensure_utc_time(_, attribute, value):
     raise ValueError(f"'{attribute.name}' must have timezone.utc")
 
 
+def get_source_name():
+    """
+    Get the value that should be used for the source of an event.
+    """
+    source = getattr(settings, "EVENT_BUS_SERVICE_NAME", None)
```

Comment:
1. I think we need setting annotations here.
2. Maybe just on the PR, and maybe in the docs, you should explain:
  - Why we need the new setting, and when we should use it over the SERVICE_VARIANT setting. I imagine you basically wish to deprecate use of SERVICE_VARIANT here?
  - Is it ok that an "EVENT_BUS" setting will be used for all events, regardless of whether or not they are sent over the event bus? It _may_ be ok, but maybe we should note this.

Not sure if we need a small ADR around this, or if we can just use code comments?

Reply:
I didn't put settings annotations here because I put them in event-bus-kafka. Not sure what the protocol is when the same setting is used in 2 different places. I can change that.
I'd indeed like to deprecate use of service variant since that only really applies to lms and cms.
I'd be fine with EVENTS_SERVICE_NAME but we do need to decide on one that we'll use both here and in EBK.

Reply:
I really don't think this needs an ADR

Reply:
[inform] @marcotuts @felipemontoya

Reply:
@mariajgrimaldi: I don't know if this will notify you twice, but I added the wrong username and had to edit it above. I'm adding this in case you never got a notification.

Reply:
The PR description explains why we shouldn't use SERVICE_VARIANT. I think it would actually be more confusing to talk about it more in the code comments.